test/utility/util_miscs.c

```diff
@@ -552,4 +552,4 @@ example_setup(int argc, char *const *argv)
         home = "WT_HOME";
     testutil_make_work_dir(home);
     return (home);
-}
+}
```

Comment:
Just a heads-up: when you see this symbol, it means the file is missing a newline at the end. If you're using `git add -p` it probably whinged, but it's easy to miss. You can fix it by adding a newline at the end of the file, or let me know if that's unclear or you need a hand.

Reply:
Ok cool I'll make those changes